### ISSUE TITLE
Question about the loss functions

### ISSUE THREAD

**Issue by ptran1203** [clovaai/WCT2#26]
Hi, thank you for your great work. I have a question about the loss function in the training phase.
In session 5.1, you have mentioned: 

> minimizing the L2 reconstruction loss and the additional feature Gram matching loss with the encoder.

So you input to network a single image x (from MS-COCO) and calculate the loss with the output is image x' as
```
Loss = recontruction_loss + gram_matching_loss
with:
+ recontruction_loss = L2(x - x')
+ gram_matching_loss = gram_matrix(encoder(x)) - gram_matrix(encoder(x'))
```

Do I miss something? Thank you.
